tests-examples/demo-todo-app.spec.ts

```diff
@@ -0,0 +1,489 @@
+import { expect, test, type Page } from '@playwright/test';
```

Comment:
We probably don't need to commit the test examples do we? Or is this being kept in temporarily for reference?